README.md

```diff
@@ -49,9 +49,9 @@ Come find us on the [development community chat](https://zulip.com/development-c
 - **Checking Zulip out**. The best way to see Zulip in action is to drop by the
   [Zulip community server](https://zulip.com/development-community/). We also
   recommend reading about Zulip's [unique
-  approach](https://zulip.com/why-zulip/) to organizing conversations.
+  approach](https://zulip.com/why-zulip/) to organize conversations.
 
-- **Running a Zulip server**. Self host Zulip directly on Ubuntu or Debian
+- **Running a Zulip server**. Self-host Zulip directly on Ubuntu or Debian
```

Comment:
I found one other instance of this type in the project; will just do that change before merging this as well.